Gemfile

```diff
@@ -114,6 +114,13 @@ group :development, :test do
   gem 'spring-commands-rspec'
 end
 
+group :development, :test do
+  gem 'rubocop',             require: false
```

Comment:
specify the version for each of these so we're all consistent
Ex: For me, the version of `rubocop` for this project is `0.75.0`,  reports an error with the `.rubocop.yml` file

Reply:
I fixed all versions in the Gemfile, for now I kept rubocop at 0.75.0:

  * rubocop (0.75.0)
  * rubocop-performance (1.6.1)
  * rubocop-rails (2.5.2)
  * rubocop-rspec (1.35.0)